### PR TITLE
add Cora Dataset

### PR DESCRIPTION
**Pull request by CarloLucibello**

### CODE REVIEW

**Review by johnnychen94 — APPROVED** [review]

File: src/Cora/Cora.jl [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Ranking" <https://arxiv.org/abs/1707.03815>`_ paper.
+Nodes represent documents and edges represent citation links.
+
+## Interface
+
+- [Cora.alldata](@ref)
+"""
+module Cora
+
+using Base: assert_havelock
```
Comment (johnnychen94) [inline src/Cora/Cora.jl:17]: Is this used?

**Review by johnnychen94 — COMMENTED** [review]
unsure what's happening to the documentation..

File: Project.toml [hunk 2]
```diff
@@ -7,6 +7,7 @@ BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
 ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
 DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```
Comment (johnnychen94) [inline Project.toml]: This shouldn't be added to `Project.toml`.

**Comment by CarloLucibello** [comment]
> unsure what's happening to the documentation..

should be fixed now

**Pull request merged**